policyengine_us/reforms/harris/rent_relief_act/rent_relief_tax_credit.py

```diff
@@ -0,0 +1,64 @@
+from policyengine_us.model_api import *
+
+
+def create_rent_relief_tax_credit() -> Reform:
+    class rent_relief_tax_credit(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "Rent Relief Tax Credit"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://www.congress.gov/bill/116th-congress/senate-bill/1106/text"
+
+        def formula(tax_unit, period, parameters):
+            gross_income = add(tax_unit, period, ["irs_gross_income"])
+            # The applicable rent amount is capped at fair market rent
```

Comment:
Do so based on a (maybe new) variable for small area fair market rent

Reply:
I think fair market rent is broken out by county for each state, we currently have it only in [Vermont](https://github.com/PolicyEngine/policyengine-us/blob/master/policyengine_us/parameters/gov/states/vt/tax/income/credits/renter/fair_market_rent.yaml) as it is a significant addition

I can create an empty input variable for now and then we can add state specific values

Reply:
small area fair market rent is different. empty variable works.